api/graphql/schema.graphql

```diff
@@ -165,10 +162,189 @@ input UpdateUserInput {
     preferredLang: Language @rename(attribute: "preferred_lang")
 }
 
+input LocalizedStringInput {
+    en: String
+    fr: String
+}
+
+input CreateUserBelongsTo {
+    connect: ID
+}
+
+input CreatePoolBelongsTo {
+    connect: ID
+}
+
+input CreateClassificationInput {
+    name: LocalizedStringInput
+    group: String!
+    level: Int!
+    minSalary: Int @rename(attribute: "min_salary")
+    maxSalary: Int @rename(attribute: "max_salary")
+}
+
+input CreateClassificationBelongsToMany {
+    create: [CreateClassificationInput!]
+    connect: [ID!]
+}
+
+input CreateCmoAssetInput {
+    key: String!
+    name: LocalizedStringInput!
+    description: LocalizedStringInput
+}
+
+input CreateCmoAssetBelongsToMany {
+    create: [CreateCmoAssetInput!]
+    connect: [ID!]
+}
+
+input CreateOperationalRequirementInput {
+    key: String
+    name: LocalizedStringInput!
+    description: LocalizedStringInput
+}
+
+input CreateOperationalRequirementBelongsToMany {
+    create: [CreateOperationalRequirementInput!]
+    connect: [ID!]
+}
+
+input CreatePoolCandidateInput {
+    pool: CreatePoolBelongsTo @belongsTo
+    user: CreateUserBelongsTo @belongsTo
+    cmoIdentifier: ID @rename(attribute: "cmo_identifier")
+    expiryDate: Date
+    isWoman: Boolean @rename(attribute: "is_woman")
+    hasDisability: Boolean @rename(attribute: "has_disability")
+    isIndigenous: Boolean @rename(attribute: "is_indigenous")
+    isVisibleMinority: Boolean @rename(attribute: "is_visible_minority")
+    hasDiploma: Boolean @rename(attribute: "has_diploma")
+    languageAbility: LanguageAbility @rename(attribute: "language_ability")
+    locationPreferences: [WorkRegion] @rename(attribute: "location_preferences")
+    acceptedOperationalRequirements: [OperationalRequirement] @belongsToMany
```

Comment:
Need to reference the OperationalRequirementBelongsToMany, CmoAssetBelongsToMany and ClassificationBelongsToMany intputs on this.